streams/src/main/java/org/apache/kafka/streams/kstream/internals/foreignkeyjoin/SubscriptionStoreReceiveProcessorSupplier.java

```diff
@@ -91,7 +91,7 @@ public void process(final Record<KO, SubscriptionWrapper<K>> record) {
                     droppedRecordsSensor.record();
                     return;
                 }
-                if (record.value().getVersion() != SubscriptionWrapper.CURRENT_VERSION) {
```

Comment:
I see this `if` condition also in `SubscriptionResolverJoinProcessorSupplier`. Does it need to be changed there as well?

Reply:
I _think_ it is not needed because the condition in `SubscriptionResolverJoinProcessorSupplier ` checks the version of `SubscriptionResponseWrapper` (opposed to `SubscriptionWrapper`), and we didn't change it.